11-lorawan/test_spec11.py

```diff
@@ -39,7 +39,7 @@ def run_lw_test(node, ttn_client, iface, dev_id):
     node.ifconfig_flag(iface, "ack_req", enable=True)
```

Comment:
This sets an ACK to request which is also captured by PKTDUMP, so the assert FALSE below fails.

Reply:
The ACK frame is not forwarded to the upper layer. It shouldn't fail I guess

Reply:
```
> PKTDUMP: data received:
~~ SNIP  0 - size:   3 byte, type: NETTYPE_UNDEF (0)
00000000  CA  8C  A8
~~ SNIP  1 - size:   9 byte, type: NETTYPE_NETIF (-1)
if_pid: 3  rssi: -32768  lqi: 0
flags: 0x0
src_l2addr: (nil)
dst_l2addr: 02
~~ PKT    -  2 snips, total size:  12 byte
```
This is what I get with the second transmission...

Reply:
that's strange... The ACK frame is simply a downlink frame with the ACK bit set, but that frame might also have information. That looks as if the server was sending more data. I will give it a look

Reply:
Note that I'm testing on a TTN3 with changes from #218